src/itest/kotlin/nl/lifely/zac/itest/SignaleringenRestServiceTest.kt

```diff
@@ -0,0 +1,276 @@
+                        "actie" to "create",
+                        "kanaal" to "zaken",
+                        "resource" to "zaakinformatieobject",
+                        "hoofdObject" to "$OPEN_ZAAK_BASE_URI/$zaakPath",
+                        "resourceUrl" to zaakInformatieObjectenUrl,
+                        "aanmaakdatum" to ZonedDateTime.now(ZoneId.of("UTC")).toString()
+                    )
+                ).toString(),
+                addAuthorizationHeader = false
+            )
+
+            Then("the response should be 'no content'") {
+                responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HttpStatusCode.NO_CONTENT_204.code()
+            }
+        }
+
+        When("a notification is sent to ZAC that a rol is updated") {
+            val response = itestHttpClient.performJSONPostRequest(
+                url = "$ZAC_API_URI/notificaties",
+                headers = Headers.headersOf(
+                    "Content-Type",
+                    "application/json",
+                    "Authorization",
+                    ItestConfiguration.OPEN_NOTIFICATIONS_API_SECRET_KEY
+                ),
+                requestBodyAsString = JSONObject(
+                    mapOf(
+                        "actie" to "create",
+                        "kanaal" to "zaken",
+                        "resource" to "rol",
+                        "hoofdObject" to "$OPEN_ZAAK_BASE_URI/$zaakPath",
+                        "resourceUrl" to zaakRollenUrl,
+                        "aanmaakdatum" to ZonedDateTime.now(ZoneId.of("UTC")).toString()
+                    )
+                ).toString(),
+                addAuthorizationHeader = false
+            )
+
+            Then("the response should be 'no content'") {
+                responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HttpStatusCode.NO_CONTENT_204.code()
+            }
+        }
+    }
+
+    Given("A zaak has been assigned") {
+        When("the list of zaken signaleringen for ZAAK_DOCUMENT_TOEGEVOEGD is requested") {
+            lateinit var responseBody: String
+
+            eventually(
+                eventuallyConfig {
+                    duration = 5.seconds
+                    interval = 500.milliseconds
+                }
+            ) {
+                val response = itestHttpClient.performGetRequest(
+                    "$ZAC_API_URI/signaleringen/zaken/ZAAK_DOCUMENT_TOEGEVOEGD"
+                )
+                response.isSuccessful shouldBe true
+                responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                responseBody.shouldBeJsonArray()
+                JSONArray(responseBody) shouldHaveSize 1
+            }
+
+            Then("""it returns the correct signaleringen""") {
```

Comment:
You really like those triple quotes don't you? ;-)

Reply:
Nah. It's just copy paste from another test with longer description.